src/lib/utils/medallia.ts

```diff
@@ -7,46 +7,107 @@ type KampyleOnsiteSdk = Window &
     vaSurvey: string
   }
 
-const vagovstagingsurveys = {
-  '/search': 20,
-  '/contact-us/virtual-agent': 26,
+const SURVEY_NUMBERS = {
```

Comment:
The Medallia setup here was quite different from what was in content-build. It was missing the subpath string matching, so I brought over the existing logic and variables here.